cblox/include/cblox/core/submap_collection_inl.h

```diff
@@ -285,20 +294,22 @@ bool SubmapCollection<SubmapType>::saveToFile(
     return false;
   }
   // Saving the submap collection header object
-  TsdfSubmapCollectionProto tsdf_submap_collection_proto;
-  getProto(&tsdf_submap_collection_proto);
+  SubmapCollectionProto submap_collection_proto;
+  getProto(&submap_collection_proto);
   // Write out the layer header.
-  if (!voxblox::utils::writeProtoMsgToStream(tsdf_submap_collection_proto,
+  if (!voxblox::utils::writeProtoMsgToStream(submap_collection_proto,
                                              &outfile)) {
     LOG(ERROR) << "Could not write submap collection header message.";
     outfile.close();
     return false;
   }
   // Saving the tsdf submaps
   for (const auto& id_submap_pair : id_to_submap_) {
-    LOG(INFO) << "Saving tsdf_submap with ID: " << id_submap_pair.first;
+    LOG(INFO) << "Saving submap with ID: " << id_submap_pair.first;
     // Saving the submap
-    (id_submap_pair.second)->saveToStream(&outfile);
+    bool success = (id_submap_pair.second)->saveToStream(&outfile);
+    if (success) {LOG(INFO) << "Saving successful";}
+    else {LOG(WARNING) << "Saving unsuccessful";}
```

Comment:
This formatting is disallowed by google style guide:

https://google.github.io/styleguide/cppguide.html#Conditionals

I'm surprised your formatter didn't pick this up... Maybe you missed this file?

Probably worth one of us running the formatter on the entire project before merging this.